fluentd/lib/in_tail/in_tail.rb

```diff
@@ -0,0 +1,1048 @@
+#
```

Comment:
This file should be either removed entirely, appear as a patch to vendored_src, are a new file to vendored_src since you cant patch it